probe_scraper/exc.py

```diff
@@ -0,0 +1,4 @@
+class ProbeScraperInvalidRequest(Exception):
+    """Exception type for returning HTTP 4XX in push mode."""
```

Comment:
It's specifically returning only `400`. Is this docstring future-proofing?

Reply:
yes